src/components/Timestamp.jsx

```diff
@@ -0,0 +1,7 @@
+function Timestamp({time}) {
```

Comment:
```suggestion
function Timestamp({ time }) {
```
Es porque soy pijo, asi se lee mejor :D